data/sql/updates/pending_db_world/rev_1606162431492361200.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606162431492361200');
+DELETE `creature`, `creature_addon`, `waypoint_data`
+FROM
+    `creature`, `creature_addon`, `waypoint_data`
```

Comment:
i have never seen a delete query like that, u sure it works?

Reply:
@BarbzYHOOL i tested this query on my MySQL 8.0.22. Example request i looked at https://stackoverflow.com/a/28621990

Reply:
@Quickson looks good to me so we don't need to delete from every single table!